### PR TITLE
docs: document new aliases policy

### PR DESCRIPTION
**Pull request by mcornella**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- Document alias changes in CONTRIBUTING file
- Add checkbox in new PR template

## Other comments:

cc @carlosala @robbyrussell @larson-carter

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - [ ] The code is mine or it's from somewhere with an MIT-compatible license.
 - [ ] The code is efficient, to the best of my ability, and does not waste computer resources.
 - [ ] The code is stable and I have tested it myself, to the best of my abilities.
+- [ ] If the code introduces new aliases, I provide a valid use case for all plugin users down below.
```
Comment (mcornella) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: @carlosala would this work?
```suggestion
- [ ] If the code introduces new aliases, I provide a valid use case for all plugin users. See CONTRIBUTING file.
```

Comment (carlosala) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Perfect for me! 👌🏻

Comment (larson-carter) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: > @carlosala would this work?
> 
> ```suggestion
> 
> - [ ] If the code introduces new aliases, I provide a valid use case for all plugin users. See CONTRIBUTING file.
> 
> ```

Just a note from me. Could we link to the file itself?

Comment (carlosala) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: The problem is that when you are writing the PR you have the raw display of markdown, and it's going to be a bit shitty to have that `[foo](https://bar.com)` syntax.

Comment (larson-carter) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Ahh very true. Didn't take that into consideration I was thinking about it being rendered for the end user.

Comment (carlosala) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: looks like we can merge @mcornella

**Pull request merged**